<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>debugWindow</class>
 <widget class="QWidget" name="debugWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SiriKali Debug Window</string>
  </property>
  <widget class="QPushButton" name="pbClear">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>430</y>
     <width>111</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>C&amp;lear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClose">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>430</y>
     <width>111</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>421</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
